tiling.js

```diff
@@ -3953,18 +3953,27 @@ function takeWindow(metaWindow, space, {navigator}) {
     if (!navigator._moving) {
         navigator._moving = [];
         signals.connectOneShot(navigator, 'destroy', () => {
-            let space = spaces.selectedSpace;
-            navigator._moving.reverse().forEach(w => {
-                w.change_workspace(space.workspace);
-                if (w.get_workspace() === space.workspace) {
-                    insertWindow(w, {existing: true});
+            let selectedSpace = spaces.selectedSpace;
+            navigator._moving.forEach(w => {
+                w.change_workspace(selectedSpace.workspace);
+                if (w.get_workspace() === selectedSpace.workspace) {
+                    insertWindow(w, { existing: true });
+
+                    // make space selectedWindow (keeps index for next insert)
+                    selectedSpace.selectedWindow = w;
                 }
             });
 
             // activate last metaWindow after taken windows inserted
-            Utils.later_add(Meta.LaterType.IDLE, () => {
-                Main.activateWindow(metaWindow);
-            });
+            let firstWindow = navigator._moving.find(v => v !== undefined);
+            if (firstWindow) {
+                Utils.later_add(Meta.LaterType.IDLE, () => {
+                    Main.activateWindow(firstWindow);
+                });
+            }
+
+            // clean up after move
+            navigator._moving = [];
```

Comment:
Hey @Lythenas, this is the function that was fixed in this PR.

The rest of the changes are simply linter formatting changes (clean up formatting!).

Reply:
We'll apply linting across modules slowly - mainly to help/ease the burdern on @jrahmatzadeh  for his reviews.  E.g. just linting `liveAltTab`, `navigator`, `tiling` for this PR.

Reply:
Yes that seems fine to me.